public/src/supported-events-policy.md

```diff
@@ -0,0 +1,48 @@
+  - The email **must** include a detailed, itemized budget for the event, including all expenses and sources of income.
+    - The budget _should_ be balanced and _should_ include a margin of at least 10% to account for unexpected costs and circumstances.
+  - If the email request is made by a Speedcubing Canada Director or Officer, the event will be recognized as a Speedcubing Canada Supported Event unless a Director raises an objection within 72 hours of the request.
+    - If an objection is raised by a Director, the event will be recognized as a Speedcubing Canada Supported Event if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of recognition.
+  - If the email request is made by individual(s) and/or group(s) who are not a Speedcubing Canada Director or Officer, the event will be recognized as a Speedcubing Canada Supported Event if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of recognition.
+
+## Organizational and Financial Requirements
+
+- If a Speedcubing Canada Supported Event is a WCA competition, it **must** follow all WCA Regulations and Policies.
+- If a Speedcubing Canada Supported Event is a WCA competition, Speedcubing Canada **must** be listed as an Organizer on the WCA website.
+- If a Speedcubing Canada Supported Event is a WCA competition, registration **must** be run via the WCA website, registration fees **must** be collected via the WCA website and registration fees **must** be directed to Speedcubing Canada’s Stripe account.
+- Any and all surplus of funds from Speedcubing Canada Supported Events **must** be returned to Speedcubing Canada’s corporate bank account(s).
+- Any and all reimbursements to individual(s) and/or group(s) for a Speedcubing Canada Supported Event **must** be made in accordance with the latest version of the [Speedcubing Canada Reimbursement Policy](https://www.speedcubingcanada.org/documents/reimbursement-policy.pdf).
+- Speedcubing Canada’s Board of Directors reserves the right to request a full financial report for a Speedcubing Canada Supported Event from the individual(s) and/or group(s) who organized the event.
+  - If requested, a full financial report **must** be returned to Speedcubing Canada’s Board of Directors within 30 days of the request.
+
+## Legal Requirements and Insurance
+
+- Any and all Speedcubing Canada Supported Events **must** follow all legal requirements of the jurisdiction in which in the event takes place.
+- Any and all Speedcubing Canada Supported Events **must** be covered by a liability insurance policy (either provided by Speedcubing Canada directly or purchased by another party) with not less than $1 million coverage that names Speedcubing Canada as an insured party.
+  - If the insurance policy is not provided by Speedcubing Canada directly, proof of insurance **must** be emailed to [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org).
```

Comment:
```suggestion
  - If the insurance policy is not provided by Speedcubing Canada directly, proof of insurance **must** be emailed to [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org) before their event is recognized as a Speedcubing Canada Supported Event.
```
Or a similar qualification.

Reply:
In most cases Speedcubing Canada will purchase the insurance directly or the organizer will purchase the insurance via the venue (most municipalities in Ontario offer this).

I am comfortable with the insurance from the latter case being submitted "not less than two weeks before the first date of the event". If they don't provide proof of insurance, we can revoke recognition, or the more reasonable option would be purchasing ourselves at that point.

Anyways, hopefully this won't be a long term problem once we have ongoing coverage.